public/apps/configuration/panels/test/__snapshots__/role-list.test.tsx.snap

```diff
@@ -1,5 +1,169 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Role list AccessError component should load access error component 1`] = `
+<Fragment>
+  <Memo()
+    config={Object {}}
+    coreStart={
+      Object {
+        "http": 1,
+      }
+    }
+    dataSourcePickerReadOnly={false}
+    navigation={Object {}}
+    params={Object {}}
+    selectedDataSource={
+      Object {
+        "id": "test",
+      }
+    }
+    setDataSource={[MockFunction]}
+  />
+  <EuiPageHeader>
+    <EuiTitle
+      size="l"
+    >
+      <h1>
+        Roles
+      </h1>
+    </EuiTitle>
+  </EuiPageHeader>
+  <EuiPageContent>
+    <EuiPageContentHeader
+      id="role-table-container"
+    >
+      <EuiPageContentHeaderSection>
+        <EuiTitle
+          size="s"
+        >
+          <h3>
+            Roles
+            <span
+              className="panel-header-count"
+            >
+               
+              (
+              0
```

Comment:
Nit: Is this supposed to be split up like this?

Reply:
auto generated by jest. so I think yes